src/decode.rs

```diff
@@ -0,0 +1,4 @@
+pub use sqlx_core::decode::*;
```

Comment:
If you would, please copy the doc-comment for the module from `sqlx-core/src/decode.rs`.